ci/e2e/terraform_provider_ocm_files/main.tf

```diff
@@ -33,7 +33,7 @@ data "aws_caller_identity" "current" {
 
 resource "ocm_cluster_rosa_classic" "rosa_sts_cluster" {
   name               = var.cluster_name
-  version            = var.openshift_version
+  version            = "openshift-v${var.openshift_version}"
```

Comment:
did you test that it's doing anything different?

Reply:
it's not working as u said

Reply:
that's not what I'm asking
did you test this change locally?